### ISSUE TITLE
V4.0 status

### ISSUE THREAD

**Issue by ili101** [adamdriscoll/selenium-powershell#132]
Hi :)
I have some Selenium code I wrote that uses the Selenium class directly. I'm planing to migrate it to this module next week and I saw you are actively working on version 4.
What is the status of the new version? 
I was thinking of using version 4 and maybe send PR if something is not working or missing.
Is it in a usable state or is it too early for that?

**Comment by itfranck** [comment]
Hi @ili101 , 

V4 is in a usable state.
Release date is planned for Oct. 4th

BUT
I broke something (oops) that cause problems when using the pipeline (eg: Get-SeElement | Get-SeHtml). 
It will be fixed within 24 hours from me writing this. 

I will reply again on this thread as soon as I pushed the fix for the aforementioned issue. 

I definitely would encourage you to use the V4 branch over the main branch as I will need to migrate any PR made on the main branch shortly after. That being said, if you were already familiar with the main branch and preferred to use it, no harm done. 

tl;dr
V4 is in a more or less usable state for the next 24 hours due to what  I did for  #117. 
V4 will be in a fully usable state again within 24h (and I'll reply to this thread to confirm)

**Comment by itfranck** [comment]
@ili101  All good again.
